src/cgns_utilities.F90

```diff
@@ -2622,7 +2622,7 @@ subroutine convertPlot3d(pFile, cFile)
     zoneLoop: do iZone=1, nZones
 
        ! Write the zone itself.
-       write(zoneName, "((a) (I5))")  'Domain.', izone
+       write(zoneName, "((a), (I5))")  'Domain.', izone
```

Comment:
Also this

Reply:
The comma makes sense to me, but are the parentheses required? I am not sure what they are doing.

Reply:
I am not familiar with file IO specifications, not really sure :/ maybe @anily knows?